src/main/java/seedu/address/model/item/Task.java

```diff
@@ -0,0 +1,56 @@
+package seedu.address.model.item;
+
+import java.io.IOException;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+/**
+ * Stub class for the task object.
+ */
+public class Task {
+    /**
+     * Enum class that contains the priority of the tasks.
+     */
+    enum Priority implements Comparable<Priority> {
+        HIGH, MEDIUM, LOW;
+    }
+
+    private boolean done;
+    private Priority priority;
+
+    public Task() {
+        done = false;
+        priority = Priority.MEDIUM;
+    }
+
+    public Priority getPriority() {
+        return this.priority;
+    }
+
+    public boolean isDone() {
+        return this.done;
+    }
+
```

Comment:
Missing toggleDone(String): method

Reply:
As this is a stub class just for the model, it will be implemented in future by @blimyj.